### PR TITLE
[zh] sync wildcard-egress-hosts http-proxy

### PR DESCRIPTION
**Pull request by my-git9**
Please provide a description for what this PR is for.

And to help us figure out who should review this PR, please 
put an X in all the areas that this PR affects.

- [ ] Ambient
- [ ] Docs
- [x] Installation
- [ ] Networking
- [ ] Performance and Scalability
- [ ] Extensions and Telemetry
- [ ] Security
- [ ] Test and Release
- [ ] User Experience
- [ ] Developer Infrastructure

### CODE REVIEW

**Comment by ericvn** [comment]
Looking at the English version of the pages, they have not been updated in the main branch, so we can cherry-pick this back to 1.19.

File: content/zh/docs/tasks/traffic-management/egress/wildcard-egress-hosts/index.md [hunk 1]
```diff
@@ -9,13 +9,18 @@ owner: istio/wg-networking-maintainers
 test: yes
 ---
 
-[控制 Egress 流量](/zh/docs/tasks/traffic-management/egress/)任务和[配置一个 Egress 网关](/zh/docs/tasks/traffic-management/egress/egress-gateway/)示例描述如何配置特定主机的 egress 流量，如：`edition.cnn.com`。本示例描述如何为通用域中的一组特定主机开启 egress 流量，譬如：`*.wikipedia.org`，无需单独配置每一台主机。
+[控制 Egress 流量](/zh/docs/tasks/traffic-management/egress/)任务和
+[配置一个 Egress 网关](/zh/docs/tasks/traffic-management/egress/egress-gateway/)示例描述如何配置特定主机的
+Egress 流量，如：`edition.cnn.com`。本示例描述如何为通用域中的一组特定主机开启 Egress 流量，
+譬如：`*.wikipedia.org`，无需单独配置每一台主机。
```
Comment (windsonsea) [inline content/zh/docs/tasks/traffic-management/egress/wildcard-egress-hosts/index.md:15]:
```suggestion
譬如 `*.wikipedia.org`，无需单独配置每一台主机。
```
`：` 会包含到这句话 `无需单独配置每一台主机`，应该可以去掉。

File: content/zh/docs/tasks/traffic-management/egress/http-proxy/index.md [hunk 2]
```diff
@@ -8,19 +8,27 @@ aliases:
...
-## 部署 HTTPS 代理{#deploy-an-https-proxy}
+## 部署 HTTPS 代理  {#deploy-an-https-proxy}
 
-本例中为了模拟传统代理，在集群内部署了一个 HTTPS 代理。此外，为了模拟在集群外运行的更真实的代理，通过代理的 IP 地址而不是 Kubernetes 服务的域名来寻址代理的 pod。本例使用的是 [squid](http://www.squid-cache.org)，但是您可以使用任何支持 HTTP CONNECT 连接的 HTTPS 代理。
+本例中为了模拟传统代理，在集群内部署了一个 HTTPS 代理。此外，为了模拟在集群外运行的更真实的代理，
+通过代理的 IP 地址而不是 Kubernetes 服务的域名来寻址代理的 Pod。本例使用的是
+[squid](http://www.squid-cache.org)，但是您可以使用任何支持 HTTP CONNECT 连接的 HTTPS 代理。
 
-1. 为 HTTPS 代理创建一个命名空间，而不标记为用于 SideCar 注入。如果没有标签，则在新命名空间中 SideCar 注入是不可用的，因此 Istio 将无法控制那里的流量。您需要在集群之外通过这种行为来模拟代理。
+1. 为 HTTPS 代理创建一个命名空间，而不标记为用于 Sidecar 注入。如果没有标签，则在新命名空间中
```
Comment (windsonsea) [inline content/zh/docs/tasks/traffic-management/egress/http-proxy/index.md:30]:
```suggestion
1. 为 HTTPS 代理创建一个命名空间，而不打上 Sidecar 注入所用的标签。如果没有此标签，则在新命名空间中
```

File: content/zh/docs/tasks/traffic-management/egress/http-proxy/index.md [hunk 3]
```diff
@@ -8,19 +8,27 @@ aliases:
...
+   SideCar 注入是不可用的，因此 Istio 将无法控制那里的流量。您需要在集群之外通过这种行为来模拟代理。
```
Comment (windsonsea) [inline content/zh/docs/tasks/traffic-management/egress/http-proxy/index.md:31]:
```suggestion
   SideCar 注入是被禁用的，因此 Istio 将无法控制这个命名空间的流量。您需要在集群之外通过这种行为来模拟代理。
```

**Review by windsonsea — APPROVED** [review]
Thanks.
/lgtm

**Pull request merged**